runtime/src/accounts_db.rs

```diff
@@ -2385,22 +2396,26 @@ impl AccountsDB {
         );
     }
 
-    fn purge_slot_cache_keys(&self, dead_slot: Slot, slot_cache: SlotCache) {
+    fn purge_slot_cache_keys(
+        &self,
+        dead_slot: Slot,
+        slot_cache: SlotCache,
+    ) -> Vec<(u64, AccountInfo)> {
```

Comment:
no need to return? (maybe incorrect rebase cleanup because I added late-comment at the AccountsDB caching pr).